test/parallel/sdk.API.utils.test.ts

```diff
@@ -93,15 +98,9 @@ test("GIVEN buyAsset WHEN operation is confirmed AND isMultiSwapAssetTransaction
 });
 
 test("GIVEN buyAsset WHEN operation is failed AND isMultiSwapAssetTransactionSuccessful THEN it returns false", async () => {
-  const [token2] = await Assets.setupUserWithCurrencies(
-    sudo,
-    [defaultCurrencyValue],
-    sudo,
-  );
-
   const buyAssetEvent = await signTx(
     api,
-    Xyk.buyAsset(token1, token2, new BN(1000)),
```

Comment:
why not set up the token & pool?  am I missing sth?

Reply:
I wanted the function to have the same parameters as the new one in the pallet, and we would have the possibility to set the wrong pool in options. I think your way would be better now, but I made too many changes to check it.

Reply:
If we have the same error I guess is fine if we have the same errors,